unix_integration/resolver/src/idprovider/kanidm.rs

```diff
@@ -88,7 +88,8 @@ impl KanidmProvider {
                 IdpError::Tpm
             })?;
 
-        let crypto_policy = CryptoPolicy::time_target(Duration::from_millis(250));
+        // let crypto_policy = CryptoPolicy::time_target(Duration::from_millis(250));
```

Comment:
This causes a memory leak? 😱

Reply:
Ahhh no, it causes a spike at startup and then it drops. It causes "bursts" of higher memory usage through out operation too during an auth. I probably should revert this bit actually ....